### PR TITLE
Replace XML Response object in GetTransportCert with a JSON object

### PR DESCRIPTION
**Pull request by ckelleyRH**
Start to replace some of the uses of the `XMLObject` class.

### CODE REVIEW

File: base/kra/src/main/java/com/netscape/cms/servlet/csadmin/GetTransportCert.java [hunk 1]
```diff
@@ -133,16 +135,20 @@ protected void process(CMSRequest cmsReq) throws EBaseException {
...
             logger.debug("GetTransportCert: Sending response " + mime64);
-            XMLObject xmlObj = new XMLObject();
-            Node root = xmlObj.createRoot("XMLResponse");
 
-            xmlObj.addItemToContainer(root, "Status", SUCCESS);
-            xmlObj.addItemToContainer(root, "TransportCert", mime64);
-            byte[] cb = xmlObj.toByteArray();
+            ObjectMapper mapper = new ObjectMapper()
+                    .enable(SerializationFeature.INDENT_OUTPUT)
+                    .setAnnotationIntrospector(new JacksonAnnotationIntrospector());
```
Comment (ckelleyRH) [inline base/kra/src/main/java/com/netscape/cms/servlet/csadmin/GetTransportCert.java:141]: I don't suppose this is necessary if we're about to convert to `byte[]` and fire it off, without printing the `JSON` anywhere.

**Review by edewata — APPROVED** [review]
LGTM. We can ignore the CI failure since it's a known issue in NSS/JSS.

**Comment by edewata** [comment]
BTW, it might be nice if we could define a response class for this and use the same `JSONSerializer` to generate the JSON response. That way we don't have to use Jackson API directly.

**Comment by ckelleyRH** [comment]
If it might help further removal that's a good idea.

**Pull request merged**